lib/gcloud/search/index.rb

```diff
@@ -13,7 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-require "gcloud/search/index/list"
+require "gcloud/search/result/list"
```

Comment:
On the other services we are requiring the model, and let the model require the list. So I think the requires here should be:

```ruby
require "gcloud/search/index/list"
require "gcloud/search/result"
```